osu.Game/Database/RealmExtensions.cs

```diff
@@ -47,5 +48,31 @@ public static class RealmExtensions
 
             return mapper.Map<T>(item);
         }
+
+        /// <summary>
+        /// Wrap a managed instance of a realm object in a <see cref="Live{T}"/>.
+        /// </summary>
+        /// <param name="item">The item to wrap.</param>
+        /// <param name="contextFactory">A factory to retrieve realm contexts from.</param>
+        /// <typeparam name="T">The type of object.</typeparam>
+        /// <returns>A wrapped instance of the provided item.</returns>
+        public static Live<T> Wrap<T>(this T item, IRealmFactory contextFactory)
+            where T : RealmObject, IHasGuidPrimaryKey => new Live<T>(item, contextFactory);
```

Comment:
As you even stated in the issue thread, I'd probably envision this being a method on `RealmContextFactory`, as the back-and-forth passing of `contextFactory` as written feels a bit off.

Reply:
This helper method no longer exists.